ports/raspberrypi/boards/datanoise_picoadk_v2/mpconfigboard.h

```diff
@@ -0,0 +1,21 @@
+// This file is part of the CircuitPython project: https://circuitpython.org
+//
+// SPDX-FileCopyrightText: Copyright (c) 2021 Scott Shawcroft for Adafruit Industries
+//
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#define MICROPY_HW_BOARD_NAME "Datanoise PicoADK V2"
+#define MICROPY_HW_MCU_NAME "rp2350a"
+
+
+#define DEFAULT_I2C_BUS_SCL (&pin_GPIO9)
+#define DEFAULT_I2C_BUS_SDA (&pin_GPIO8)
+
+#define DEFAULT_SPI_BUS_SCK (&pin_GPIO18)
+#define DEFAULT_SPI_BUS_MOSI (&pin_GPIO12)
+#define DEFAULT_SPI_BUS_MISO (&pin_GPIO20)
```

Comment:
The pins here don't match `board.SCK` etc  in `pins.c`, and the MOSI choice is for SPI1, not SPI0. Did you mean:
```suggestion
#define DEFAULT_SPI_BUS_SCK (&pin_GPIO6)
#define DEFAULT_SPI_BUS_MOSI (&pin_GPIO3)
#define DEFAULT_SPI_BUS_MISO (&pin_GPIO4)
```